### DISCUSSION TITLE
Fly Plugin Compatible with griefprevention!

### DISCUSSION THREAD

**Discussion by sebastianjnuwu** [GriefPrevention/GriefPrevention#1878] [Q&A] [answered]
Hello, would someone have a plugin like that that allows the player to fly inside their Clains? I've seen some more so discontinued if vices know a simple one that they take thank you

**Comment by sebastianjnuwu** [comment, marked as answer]
if anyone knows, tag me! Thanks in advance!

**Reply by QarthO** [reply]
checkout gpflags
